src/helpers/config.ts

```diff
@@ -69,3 +78,18 @@ export async function extractChainId(conn: Config = {}): Promise<number> {
 
   return chainId;
 }
+
+export function jsonFileAliases(filepath: string): AliasesNameMap {
```

Comment:
This is really more of an example than part of the SDK, but it seems like a very common use-case and should be useful to get new devs up and running.
It's only useful in the Node.js environment.

Reply:
I like it!